### DISCUSSION TITLE
Vulnérabilités En Javascript

### DISCUSSION THREAD

**Discussion by Shygeek64** [community/community#134289] [Codespaces]
### Select Topic Area

Question

### Body

Voilà mon désarroi: J'ai cru tout perdre mes Vulnérabilités avant de pouvoir les corriger, voici un aperçu: 

<!DOCTYPE HTML>
<html lang="fr">
<head>
    <meta charset="UTF-8">
    <meta HTTP-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width-device-width, initial-scale="1.0">
    <title>Document</title>
    <script src="script.js">let numberCve = prompt(CVE-2022-38375)</script>
</head>
<body>
    <header id="mainheader">
        {"color="red"}
        {"bgcolor="midnight-blue"}
        {"font-size="30%"}
        {"align="center"}
        {"text="strong"}
        {"police-text="time-new-roman"}

        <p>Vector String CVSS: 3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
            Severity Score CVSS: Critical 9.8
           Vector String CVSS: 3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:H
            Severity Score CVSS: Critical 9.1 By timidhacker@hackerone.com</p>
            <h1>Name Of CWE: Improper Authorization</h1>
            <h2>CWE ID: CWE-285</h2>
            <h3>Asset's Name Or Bug Bounty's Prime: IP Adderss For $600 000,00</h3>
            <h4>EPSS: 0,37%</h4>
            <h5>NVD Published Date: 16/02/2023</h5>
            <h6>NVD Last Modified: 06/11/2023</h6>
            <h7>Reported By Fortinet, Inc.</h7>
            <h8>Programmed By HackerOne</h8>
            <h9>To Share With NIST</h9>
        </header>
    </body>
</html>

**Comment by Shygeek64** [comment]
Je n'ai pas la première partie du code mais je vais reprendre

**Comment by Shygeek64** [comment]
<!DOCTYPE HTML>
<html lang="fr">
<head>
    <meta charset="UTF-8">
    <meta HTTP-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width-device-width, initial-scale="1.0">
    <title>Document</title>
    <script src="script.js">let numberCve = prompt(CVE-2022-38375)</script>
</head>
<body>
    <header id="mainheader">
        {"color="red"}
        {"bgcolor="midnight-blue"}
        {"font-size="30%"}
        {"align="center"}
        {"text="strong"}
        {"police-text="time-new-roman"}

        <p>Vector String CVSS: 3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H
            Severity Score CVSS: Critical 9.8
           Vector String CVSS: 3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:H
            Severity Score CVSS: Critical 9.1 By timidhacker@hackerone.com</p>
            <h1>Name Of CWE: Improper Authorization</h1>
            <h2>CWE ID: CWE-285</h2>
            <h3>Asset's Name Or Bug Bounty's Prime: IP Adderss For $600 000,00</h3>
            <h4>EPSS: 0,37%</h4>
            <h5>NVD Published Date: 16/02/2023</h5>
            <h6>NVD Last Modified: 06/11/2023</h6>
            <h7>Reported By Fortinet, Inc.</h7>
            <h8>Programmed By HackerOne</h8>
            <h9>To Share With NIST</h9>
        </header>
    </body>
</html>

**Comment by Shygeek64** [comment]
Je ne comprends pas pourquoi ça ne fonctionne pas ;(

**Comment by Akash1134** [comment]
Hi @Shygeek64 , thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
